components/nimbus/src/metrics.rs

```diff
@@ -0,0 +1,59 @@
+use crate::{
+    enrollment::ExperimentEnrollment, glean_metrics::nimbus_events::EnrollmentStatusExtra,
+    EnrollmentStatus,
+};
+use serde_derive::{Deserialize, Serialize};
+
+pub trait HostMetricsHandler: Send + Sync {
+    fn record_enrollment_status(&self, enrollment_status_extra: EnrollmentStatusExtraDef);
+
+    /// Only used for testing
```

Comment:
I was hoping to use `#[cfg(tests)]` but UniFFI didn't like that. I confirmed with bdk that there wasn't much that could be done other than comment that it's just used for testing.

Reply:
Realization - I can get rid of these by using the actual glean client in its testing mode. That's coming in a little while, sorry